vuepress/docs/next/docs/reference/entando-cli.md

```diff
@@ -71,23 +71,23 @@ Use `ent help` to review the list of available commands
   - Dectivate using: ~/.entando/ent/v6.3.2/cli/v6.3.2/deactivate
 
 > Available commands:
-  - app                  => Helps managing an EntandoApp
-  - app-info             => Displays information about an entando app
-  - bundler              => Wrapper for the ENT private installation of the entando bundle tool.
+  - app                  => Helper for managing an Entando App
```

Comment:
So, this is verbatim from ent --help. What you have reads better but I was thinking we should try to keep it in sync. @w-caffiero-entando, thoughts?

Reply:
Let's go with this and I'll open an ECS ticket.